lib/structuraid_core/engineering/analysis/footing/utils/shear_moment.rb

```diff
@@ -0,0 +1,87 @@
+module StructuraidCore
+  module Engineering
+    module Analysis
+      module Footing
+        module Utils
+          module ShearMoment
```

Comment:
I think this name signals an opportunity for a clearer separation of concerns in this module. What about having two modules called `OneWayShear` and `OneWayBendingMoment` instead of this one?